### ISSUE TITLE
Record only the Text Chat?

### ISSUE THREAD

**Issue by Powerbock** [horacio9a/chaturbate-anonymous#10]
I am looking for a technical possibility to automatically save the pure chat history, not only the video as a file, but also the messages of the chat into a text file e.g. .  Would this also be possible?

**Comment by hico700** [comment]
yep, that would be absolutely great, but I have no idea if it could be done ... :-)  I just tried to use a screen recorder, but it's not comfortable too much :-(

**Comment by RoyalFlyBy** [comment]
Is there a substantial amount of people that want this type of feature because I can add this rather easily as I already reverse engineered the chat for an unofficial chatbot.

If implemented how would you want the data to be represented, a list of messages like:
```
[time] user: message
[time] user: message
```

or in JSON:
```
[
    {
        time: TIME,
        user: USER,
        msg: MESSAGE
    }, 
    {
        time: TIME,
        user: USER,
        msg: MESSAGE
    }
]
```
Or differently, in which case I would need an example.

**Comment by hico700** [comment]
It is great to hear that you can read and save the messages from the chat. I think it will be best to save all the chat content inc. tips. It will be the right way, because the video is many times related to the tip value (for ex. lush vibs) and the information explains what's going on in the video (or why)... :-)   Maybe a little problem will be with a synchronization on the video, if there are a time drop outs...  ???
